akkeeper-yarn/src/main/scala/akkeeper/yarn/LocalResourceNames.scala

```diff
@@ -19,6 +19,7 @@ private[akkeeper] object LocalResourceNames {
   val AkkeeperJarName = "akkeeper.jar"
   val UserJarName = "user.jar"
   val UserConfigName = "user_config.conf"
+  val LauncherConfigName = "launcher_config.conf"
```

Comment:
I wouldn't say it's a launcher config per se. Morel like `application` or `akkeeper` config.

Reply:
Renamed to `application.conf`